theseus/geometry/tests/test_so3.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest  # noqa: F401
+import torch
+
+import theseus as th
+
+from .common import (  # check_inverse,; check_compose,; check_exp_map,; check_log_map,
```

Comment:
Remove commented code.

Reply:
I will remove these commented code at the end. This is just used to remind me future tests :)